packages/blaze-emulator/src/emulator.ts

```diff
@@ -504,7 +504,7 @@ export class Emulator {
 
     // Minimum collateral amount included
     const minCollateral = BigInt(
-      Math.ceil(this.params.collateralPercentage * Number(body.fee())),
+      Math.ceil((this.params.collateralPercentage * Number(body.fee())) / 100),
```

Comment:
@micahkendall I'm pretty sure this should be `Math.ceil((this.params.collateralPercentage / 100) * Number(body.fee()))` if we're trying to keep the notion of `150 / 100`. This would also be true for the other instance that uses this parameter.

We had to revert to 0.6.0 because 0.6.1 was breaking all of our collateral values.

Reply:
PR submitted here: https://github.com/butaneprotocol/blaze-cardano/pull/184